package/freescale-imx/firmware-imx/firmware-imx.hash

```diff
@@ -1,5 +1,5 @@
 # Locally calculated
-sha256  937e196476b8e95b4b7f2501a14c8326d8a0649f8a3f9228b72373770a08deb3  firmware-imx-8.15.bin
-sha256  35188e65dbb9c7da4bbcb77c7726e835607f9f41b8b44149806ea51429ca9a31  COPYING
-sha256  30f61825583b4c26d29a798ad7e4c8ef2f2f390b1e964af302d2dc40e93cb0a4  EULA
+sha256  c3447f0f813415ccea9dc2ef12080cb3ac8bbc0c67392a74fc7d59205eb5a672  firmware-imx-8.21.bin
+sha256  de37a0bcbf1717b910c1a53ea6eab853c404e61e8143bb6c081d39f532571e54  COPYING
+sha256  b158adaac5e2d1c7fa6e739ff168ab498957e3303070f1c0b0e66d9a8f02a354  EULA
```

Comment:
cool - you'll want to make sure `make legal-info` still works - actually the GitHub actions should do this so check with @nickgudman-cpi since we want the hashes to work on the legal-info build.